cicecore/cicedynB/infrastructure/io/io_netcdf/ice_restart.F90

```diff
@@ -228,9 +227,7 @@ subroutine init_restart_write(filename_spec)
          call define_rest_field(ncid,'uvel',dims)
          call define_rest_field(ncid,'vvel',dims)
 
-#ifdef CESMCOUPLED
```

Comment:
Adding coszen to the restart files replicates EMC_CICE5 behaviour.

Reply:
Great news. We will still need some sort of ifdef around this.

Reply:
In the EMC CICE5, there was originally no coszen present as a restart field so I didn't realize this is a coupled-only field. There is no NEMSCOUPLED; do we need one or could we use the exisiting "coupled" ifdef?